odoorpc/rpc/__init__.py

```diff
@@ -38,6 +38,15 @@ def __init__(self, host, port=8069, timeout=120, version=None):
             self.port = int(port)
         self._timeout = timeout
         self.version = version
+        # Default autoretry for .odoo.com (saas) hosts
+        if 'autoretry' in kwargs:
+            self._autoretry = kwargs['autoretry']
+        elif host.endswith('.odoo.com'):
+            self._autoretry = True
+        else:
+            self._autoretry = False
```

Comment:
I don't particularly like this kind of magic. But in case it were, then you can default the `autoretry` arg to `None`, enabling this magic, insteado of relying in the existence of the kwarg. It would be the same result, but simplify the methods signature.

Making things obvious and predictable is good IMHO.

In any case, this (and the rest of the feature) deserves an entry in the docs.